### ISSUE TITLE
Relationship between device channels and channels returned by `get_device_list_usage`

### ISSUE THREAD

**Issue by waldner** [magico13/PyEmVue#82]
For my device (solar plant), if I dump `device.channels` data I get a list of names/channel number as follows:

```
name: None
channel_num: 1,2,3
channel_multiplier: 1.0
channel_type_gid: None
-------
name: Inverter 1
channel_num: 1
channel_multiplier: 2.0
channel_type_gid: 13
-------
name: Inverter 2
channel_num: 2
channel_multiplier: 2.0
channel_type_gid: 13
-------
name: None
channel_num: 3
channel_multiplier: 1.0
channel_type_gid: None
-------
[cut 12 equal channels...]
...
device_gid: xxxxxx
name: None
channel_num: 16
channel_multiplier: 1.0
channel_type_gid: None
-------
```

However, the result of the call to `get_device_list_usage` contains a rather different set of channels:

```
name: Main
channel_num: 1,2,3
-------
name: Inverter 1
channel_num: 1
-------
name: Inverter 2
channel_num: 2
-------
name: TotalUsage
channel_num: TotalUsage
-------
name: Balance
channel_num: Balance
-------
```

So why are these sets different? (it looks like `TotalUsage` usage always equals the sum of `Main`, `1` and `2`, while `Balance` usage usually equals `TotalUsage`).

Also, how can I call `get_chart_usage()` on the `TotalUsage` and `Balance` channels to get a timeseries, if that is possible at all?

**Comment by waldner** [comment]
So it looks like I was not really understanding the values correctly. `get_device_list_usage` and `get_chart_usage` contain basically (in my case, at least) the same data, which can be confirmed by calling `get_device_list_usage` passing the `instant` argument to request the values at a given time, and checking that the returned values match those in the chart usage series at the same timestamp.
As for the computed values (eg `TotalUsage`) that only appear in the output of `get_device_list_usage`, they can be calculated using other values that _do_ appear, so new (calculated) series can be built that would contain the values that would be returned by a hypothetical call to `get_chart_usage` for that channel.

In short, it looks like I don't really need `get_device_list_usage`, but your explanation helped my understanding. Thanks!

**Comment by magico13** [comment]
The biggest difference is if you need all the channels at once or if you need the history for a single channel. If you've got a bunch of devices/channels, for instance I have about 30, I'm generally using the call to get all of them at one instant, rather than looping through all of them. If you're just really watching one or two channels then the history might be more useful.

**Comment by waldner** [comment]
Well, the history call gives me the data for all channels (ie contains multiple series), so it seems I'm getting the best of both worlds with that (because as I said I can compute other values out of these). However I only have 2-4 channels, I don't know what happens if you have 30.

**Comment by waldner** [comment]
Ok the history call does NOT give me data for all channels, only for the one specified in the arguments; what I meant is that I can call the history method for each channel I'm interested into.
